src/service/storage-impl.js

```diff
@@ -82,14 +82,25 @@ export class Storage {
   }
 
   /**
-   * Saves the value of the specified property. Returns the promise that's
-   * resolved when the operation completes.
+   * Saves the value (restricted to boolean value) of the specified property.
+   * Returns the promise that's resolved when the operation completes.
    * @param {string} name
    * @param {*} value
    * @return {!Promise}
    */
   set(name, value) {
     dev().assert(typeof value == 'boolean', 'Only boolean values accepted');
+    return this.setNonBoolean(name, value);
+  }
+
+  /**
+   * Saves the value of the specified property. Returns the promise that's
+   * resolved when the operation completes.
+   * @param {string} name
+   * @param {*} value
+   * @return {!Promise}
+   */
+  setNonBoolean(name, value) {
```

Comment:
are we separating it out only for the purpose of presubmit check?

Reply:
yes

Reply:
If we already review the usage of storage service in general, I don't see too much value to add another presubmit check for non-boolean value.

Reply:
I asked before why we have the boolean value check, and the answer was that we don't want to run privacy review for every value stored. Which means that the non boolean value requires more strict privacy check and I believe another presubmit check is necessary.

Reply:
Got you. Could you indicate that in the JS doc that we need stricter privacy check for non-boolean storage.

Reply:
done